clients/fides-js/src/components/LanguageSelector.tsx

```diff
@@ -43,7 +43,7 @@ const LanguageSelector = ({ i18n, options }: LanguageSelectorProps) => {
       <div className="fides-i18n-pseudo-button">
         <svg
           xmlns="http://www.w3.org/2000/svg"
-          height="inherit"
+          height="100%"
```

Comment:
Ooh, a bonus fix... I think I saw console errors for this?

Reply:
yep! my bad